test/quantization/core/test_quantized_tensor.py

```diff
@@ -140,6 +140,48 @@ def _compress_uniform_simplified(X, bit_rate, xmin, xmax, fp16_scale_bias=True):
     return Xq, loss
 
 class TestQuantizedTensor(TestCase):
+    def test_qtensor_to_memory_format(self):
```

Comment:
looks like we are testing multiple things here, maybe it would make sense to split this test into multiple tests